### DISCUSSION TITLE
How to change the default HDR env inside the Viewer

### DISCUSSION THREAD

**Discussion by digityze2023** [google/model-viewer#4156] [Q&A] [answered]
Hello, I'm going to create a custom viewer in order to validate my GLB models. I want to create a new one with my custom HDR - I don't need to change it or other things. I started to modify the space-opera.js file but I don't know how to change the default HDR as I told you, could you please help me?

**Comment by elalish** [comment]
By viewer, I'm guessing you mean our editor? You can just drag and drop your HDR there. For `<model-viewer>`, just set `environment-image` to your HDR.

**Reply by digityze2023** [reply]
Hello, thank you for the answer. I'm customizing the Editor for my needs. Editor is based on space-opera.js library and I am working on it.

My question is: is it possible to create a custom viewer with my preferred HDR? So there will be an empty page where I can drag and drop my GLB and see it. Without having the actual one Editor setting on the right and so on.

**Reply by elalish** [reply, marked as answer]
Yes, as I said, just set `environment-image` in model-viewer. If you search that string in space-opera, you'll see where it's currently set and you can change it.
